build.sbt

```diff
@@ -16,7 +16,9 @@ scalacOptions := Seq(
 libraryDependencies ++= Seq(
   "com.azavea"                  %% "scaliper"              % "0.5.0-SNAPSHOT" % "test",
   "io.dylemma"                  %% "xml-spac"              % "0.3-SNAPSHOT",  // Published locally
-  "org.apache.spark"            %% "spark-core"            % "2.1.0",  // TODO add "provided" later
+  "org.apache.spark"            %% "spark-core"            % "2.2.0",  // TODO add "provided" later
```

Comment:
You can use

```
fullClasspath in Runtime := (fullClasspath in (Compile, run)).value
```

in the build file to allow for these to be marked as `provided` while still allowing `sbt run`